aio/content/examples/animations/src/app/hero-list-page.component.css

```
@@ -1,51 +1,33 @@
 .heroes {
-  margin: 0 0 2em 0;
   list-style-type: none;
   padding: 0;
-  width: 15em;
 }
 
 .heroes li {
-  position: relative;
-  height: 2.3em;
   overflow:hidden;
-  margin: .5em;
+  margin: .5em 0;
 }
 
 .heroes li > .inner {
   cursor: pointer;
   background-color: #EEE;
-  padding: .3em 0;
-  height: 1.6em;
+  padding: .3rem 0;
+  height: 1.6rem;
   border-radius: 4px;
-  width: 19em;
 }
 
 .heroes li:hover > .inner {
-  color: #607D8B;
+  color: black;
   background-color: #DDD;
   transform: translateX(.1em);
 }
 
-.heroes a {
-  color: #888;
-  text-decoration: none;
-  position: relative;
-  display: block;
-  width: 250px;
-}
-
-.heroes a:hover {
-  color:#607D8B;
-}
-
 .heroes .badge {
   display: inline-block;
   font-size: small;
   color: white;
   padding: 0.8em 0.7em 0 0.7em;
-  background-color: #607D8B;
-  line-height: 1em;
+  background-color: #3D5157;
```

Super-nit (to be consistent with other examples): 

```suggestion
  background-color: #3d5157;
```